mlir/test/Target/LLVMIR/Import/vscale.ll

```diff
@@ -0,0 +1,10 @@
+; RUN: mlir-translate -import-llvm -split-input-file %s | FileCheck %s
+
+define void @vscale_func() #0 {
+; CHECK: llvm.func @vscale_func()
+; CHECK-SAME: vscale_range(2, 8)
+  ret void
+}
+
+attributes #0 = { vscale_range(2,8) }
```

Comment:
nit: can you inline the attribute right in the function definition to avoid the extra layer of indirection.